### ISSUE TITLE
03-CUDA10 Docker: Is there a reason to import duplicate videos?

### ISSUE THREAD

**Issue by mihsamusev** [ahrnbom/strudl#16]
Hi,
When the 03-CUDA10 Docker STRUDL server is on On POST requests to `/datasets`, `/datasets/masks`, `/datasets/config` the files like `config.txt`, `classes.json` and `mask.png` are overwritten if the same POST request is submitted multiple times.

However on `/jobs/import_videos` the original videos and logs get a random 4 digit addition to its name at every POST request, which submits duplicate videos and logs to the server.

Is there a specific reason for that?

**Comment by ahrnbom** [comment]
I can't think of any situation where this would be useful. But I also don't really see why you would want to call it twice anyway, and the reason for doing so probably affects the desired behaviour. For example, if you've changed the config so that the newly imported videos will get a new resolution, then the old videos should probably be removed first anyway. So an option to remove any previous existing videos on an import would make sense, if that's something that people want to do.

**Comment by mihsamusev** [comment]
Yes, removing feature for datasets and videos would be very useful. The reason i'm asking is that i'm creating a script to automate dataset creation. The script will analyze my internal project folder with files and settings to post dataset, config, mask, and videos, all in one go. In this way once project folder is arranged well we are just 1 script run away from starting the annotation proccess.
So i was trying to see, should i prevent my script from posting a request to import videos that are already imported or STRUDL API lets you overwrite them (like with dataset, config, mask).
